### DISCUSSION TITLE
Page Not Found - Demo DataTable Crud

### DISCUSSION THREAD

**Discussion by apanzani** [primefaces/community#2217] [PrimeFaces] [answered]
Hi, 
I'm using the demo web site ( https://www.primefaces.org/showcase/ui/data/datatable/crud.xhtml?jfwid=5b361 )
to undrestand how to create a new Product and then reload the table of products in the first page.
When I click on button "NEW" I receive a "Page Not Found" with the button "Go to Homepage".

Please, can someone check this behaviour?

Thank you a lot.
![not_found](https://github.com/primefaces/community/assets/99031405/db20ee2a-35b2-4483-8e6c-99c89a727c5e)
![crud_new](https://github.com/primefaces/community/assets/99031405/ecd1c1bd-7ac2-455d-83d0-21a50f63d499)

**Comment by melloware** [comment, marked as answer]
@apanzani this has been reported: https://github.com/primefaces/primefaces/issues/12164

But you can see a working version here: https://quarkus-faces-melloware-8a6a34c1.koyeb.app/ui/data/datatable/crud.xhtml
